.github/workflows/checks.yml

```diff
@@ -84,6 +85,7 @@ jobs:
           ["3.8", "3.8-slim-buster", ""],
           ["3.9", "3.9-slim-buster", ""],
           ["3.10", "3.10-slim-buster", "latest"],
+          ["3.11", "3.11-slim-buster", "latest"],
```

Comment:
I think we want to use the bare "latest" tag only once. So I think it should show up only one time here, for the most recent Python version.

Reply:
Understood, I will update it shortly.